io/pci/dlpar.py

```diff
@@ -88,6 +88,12 @@ def setUp(self):
         if not self.server:
             self.cancel("Managed System not got")
         self.lpar_2 = self.params.get("lpar_2", '*', default=None)
+        cmd = 'lshwres -r io -m %s --rsubtype slot --filter lpar_names=%s ' \
+              '-F drc_index,lpar_id,drc_name,bus_id' % (self.server,
```

Comment:
displaying only lpar_id is sufficient here.

Reply:
ok